server.js

```diff
@@ -0,0 +1,110 @@
+const http = require('http')
+const port = 3000
+const ip = 'localhost'
```

Comment:
costuma ser uma péssima prática forçar o `localhost` pois isso impede de funcionar em outros ambientes (p.ex. docker). Geralmente o mais recomendado seria `0.0.0.0`